.github/workflows/update-test.yaml

```diff
@@ -9,16 +9,19 @@ on:
   pull_request:
 jobs:
   update_test:
-    name: Update test PG${{ matrix.pg }}
+    name: Update test PG${{ matrix.pg }} ${{ matrix.kind }}
     runs-on: 'ubuntu-18.04'
     strategy:
       matrix:
         pg: ["11.10","12.5"]
+        opt: ["", "-r"]
```

Comment:
I think restricting the repair tests to upgrades between 1.7.xx -> 2.0 , 2.0.1 is  sufficient and we don't need to run this for the entire matrix.  I don't mind deferring that change to a later PR.  But do think it is necessary  so that update tests don't unnecessarily consume too much time.

Reply:
Agree. I picked this as an easy solution for now. We can tune it over time and should probably also avoid running a full test when we try the repair.

Reply:
Also note that the actual breaking of the dimension slices only happen if you upgrade from pre-2.0. There are checks in the SQL files that control this.